### PR TITLE
docker cmd enhancements

### PR DESCRIPTION
**Pull request by avimora**

### CODE REVIEW

**Comment by avimora** [comment]
@meirwah please review & merge

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+target/
```
Comment (meirwah) [inline .gitignore]: did you really planned to commit these???

Comment (orius123) [inline .gitignore]: what's the problem? 
we don't have gitignore right now.

Comment (meirwah) [inline .gitignore]: ok

**Comment by meirwah** [comment]
:+1:

**Pull request merged**